src/main/java/com/wcc/platform/domain/cms/pages/mentorship/StudyGroupSection.java

```diff
@@ -0,0 +1,9 @@
+package com.wcc.platform.domain.cms.pages.mentorship;
+
+import com.wcc.platform.domain.platform.StudyGroup;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotEmpty;
+import java.util.List;
+
+/** Record for the section of study groups in the Study Groups page. */
+public record StudyGroupSection(@NotBlank String title, @NotEmpty List<StudyGroup> studyGroups) {}
```

Comment:
This new class is not necessary, as you can take advantage of the generic ones: `ListSection<StudyGroup>`